### ISSUE TITLE
[Accessibility][Supporting the platform-Breadcrumb]: Visual focus indicator is not visible on more options menu's in high contrast black/white modes

### ISSUE THREAD

**Issue by v-doku** [OfficeDev/office-ui-fabric-react#9681]
Detailed description of this issue can be found here [Sharepoint](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/EQ2hhHtdeERIi6xORuBhBCAB4E7rh1br26nk886S5v-Idg?e=vABofb)

Component:Breadcrumb

**Comment by khmakoto** [comment]
Hi @v-doku, it seems that the Sharepoint link is incorrect since its the same as the one in #9657, which deals with something different from the issue described here. Can you update the link? Thanks!

**Comment by dollykumari5** [comment]
Hi @khmakoto ,  Please refer to the bug 1 in the given SharePoint [link.](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/EQ2hhHtdeERIi6xORuBhBCAB4E7rh1br26nk886S5v-Idg?e=vABofb)
